homework2/Trie.cs

```diff
@@ -0,0 +1,123 @@
+﻿namespace homework2;
+/// <summary>
```

Comment:
```suggestion
﻿namespace homework2;
﻿
/// <summary>
```
Отдельные крупные элементы файла разделяются пустыми строками, чтобы всё не мешалось в кучу.